### DISCUSSION TITLE
Accessing App Menus

### DISCUSSION THREAD

**Discussion by FelixKratz** [FelixKratz/SketchyBar#461] [Q&A] [answered]
A question I have seen come up frequently is how to access the macOS app menus when sketchybar takes the place of the macOS bar.

**Comment by FelixKratz** [comment, marked as answer]
My personal solution for this is to use a tool called [MenuWhere](https://manytricks.com/menuwhere/).

**Comment by erics118** [comment]
[Raycast](https://www.raycast.com) also has a command called `Search Menu Items`, which does basically the same thing.

**Comment by zhuoqun-chen** [comment]
Here is another one [Paletro](https://appmakes.io/paletro)🙂

**Comment by Sinjhin** [comment]
> A question I have seen come up frequently is how to access the macOS app menus when sketchybar takes the place of the macOS bar.

Think I just solved this one. It's a tad slow due to how I needed to do it, but: [Repo](https://github.com/Sinjhin/SketchyMenu)

[Comment on plugins thread](https://github.com/FelixKratz/SketchyBar/discussions/12?sort=new#discussioncomment-14467279)

Full menu parity to two levels deep with actual clickable items and anything more than two levels it just opens the native menu with or without the native bar hidden.
